tests/e2e/default/enketo/training-cards.wdio-spec.js

```diff
@@ -69,7 +70,8 @@ describe('Training Cards', () => {
 
     const confirmMessage = await trainingCardsPage.quitTraining();
     expect(confirmMessage.body).to.contain(
-      'This training is not finished. You will lose your progress if you leave now. Are you sure you want to leave?'
+      'This training is not finished. ' +
+      'If you leave now, you will lose your progress and be prompted again later to complete it.'
```

Comment:
Wondering if we could extract this text as a constant since it is quite long and used more than once?